### DISCUSSION TITLE
Not working with Apple Silicon MacOs 13.4 Ventura

### DISCUSSION THREAD

**Discussion by Tushar1337** [input-leap/input-leap#2059] [General]
<img width="1157" alt="image" src="https://github.com/user-attachments/assets/e24c5807-80d6-45fb-94d5-795fb6087ac0">

**Comment by LaissonsDire** [comment]
Same for me. 
Apple Silicon 3.x and up available as of today dont work on Ventura...Strange.

**Comment by LaissonsDire** [comment]
I tried the x86 releases and they seem to work (more or less).
